### DISCUSSION TITLE
"Discard" is "Restore"?

### DISCUSSION THREAD

**Discussion by superDuperCyberTechno** [altsem/gitu#285] [Q&A]
As far as I know, the term "discard" isn't used in Git. Experimenting with Gitu, it seems that the command "Discard" is the same as the newly added "Restore" command (which is shorthand for `git checkout -- <file>`).

Would that be correct? Wouldn't it be more intuitive to rename "Discard" to "Restore"?

**Comment by altsem** [comment]
Hi!

It's an abstraction done in Magit. Basically it undo:s things currently selected, and the command that is being run depends on what is selected.

e.g. Discard on a not-checked-in file will do a `git clean --force *file*`, or discarding a renaming would run `git mv --force *new file* *old file*`.
